examples/multiwindow.rs

```diff
@@ -0,0 +1,62 @@
+use raw_gl_context::{GlConfig, GlContext};
+
+use winit::event_loop::{ControlFlow, EventLoop};
+use winit::window::WindowBuilder;
+
+fn main() {
+    let event_loop = EventLoop::new();
+    let window = WindowBuilder::new().build(&event_loop).unwrap();
+    let window2 = WindowBuilder::new().build(&event_loop).unwrap();
+
+    let context = GlContext::create(&window, GlConfig::default(), None).unwrap();
+    let context2 = GlContext::create(&window2, GlConfig::default(), Some(&context)).unwrap();
```

Comment:
This example demonstrates multiple contexts. OpenGL seems to have a strange way of handling this.. have to create the first context and then create the second one telling it to "share" resources with the first.